libsolidity/lsp/LanguageServer.cpp

```diff
@@ -101,14 +101,29 @@ Json::Value LanguageServer::toJson(SourceLocation const& _location)
 void LanguageServer::changeConfiguration(Json::Value const& _settings)
 {
 	m_settingsObject = _settings;
+	Json::Value jsonIncludePaths = _settings["include-paths"];
+	if (jsonIncludePaths && jsonIncludePaths.isArray())
+	{
+		std::vector<boost::filesystem::path> includePaths;
+		for (Json::Value jsonPath: jsonIncludePaths)
```

Comment:
Shouldn't this be a reference? Or does it actually dispense copies?
```suggestion
		for (Json::Value const& jsonPath: jsonIncludePaths)
```